src/test/java/uk/gov/hmcts/reform/et/syaapi/service/CaseDocumentServiceTest.java

```diff
@@ -93,14 +94,21 @@ class CaseDocumentServiceTest {
         + "\"claim-submit.png\",\"_links\":{\"self\":{\"href\": \"" + MOCK_HREF_MALFORMED + "\"}}}]}";
     private static final String MOCK_RESPONSE_WITHOUT_SELF = RESPONSE_BODY
         + "\"claim-submit.png\",\"_links\":{}}]}";
+    private final String fullJsonResponse = ResourceUtil.resourceAsString(
+        "responses/caseDocumentUpload.json"
+    );
 
     private CaseDocumentService caseDocumentService;
     private MockRestServiceServer mockServer;
 
+    CaseDocumentServiceTest() throws IOException {
+        // constructor for case document series test
+    }
```

Comment:
Surprises me you need this???

Reply:
It is required for  ResourceUtil.resourceAsString() statement